test/e2e/generated_clientset.go

```diff
@@ -187,6 +188,124 @@ var _ = framework.KubeDescribe("Generated release_1_5 clientset", func() {
 	})
 })
 
+func observeJobCreation(w watch.Interface) {
+	select {
+	case event, _ := <-w.ResultChan():
+		if event.Type != watch.Added {
+			framework.Failf("Failed to observe scheduledJob creation: %v", event)
+		}
+	case <-time.After(5 * time.Minute):
+		framework.Failf("Timeout while waiting for scheduledJob creation")
+	}
+}
+
+func newTestingScheduledJob(name string, value string) *v2alpha1.ScheduledJob {
+	parallelism := int32(1)
+	completions := int32(1)
+	return &v2alpha1.ScheduledJob{
+		ObjectMeta: v1.ObjectMeta{
+			Name: name,
+			Labels: map[string]string{
+				"time": value,
+			},
+		},
+		Spec: v2alpha1.ScheduledJobSpec{
+			Schedule:          "*/1 * * * ?",
+			ConcurrencyPolicy: v2alpha1.AllowConcurrent,
+			JobTemplate: v2alpha1.JobTemplateSpec{
+				Spec: v2alpha1.JobSpec{
+					Parallelism: &parallelism,
+					Completions: &completions,
+					Template: v1.PodTemplateSpec{
+						Spec: v1.PodSpec{
+							RestartPolicy: v1.RestartPolicyOnFailure,
+							Volumes: []v1.Volume{
+								{
+									Name: "data",
+									VolumeSource: v1.VolumeSource{
+										EmptyDir: &v1.EmptyDirVolumeSource{},
+									},
+								},
+							},
+							Containers: []v1.Container{
+								{
+									Name:  "c",
+									Image: "gcr.io/google_containers/busybox:1.24",
+									VolumeMounts: []v1.VolumeMount{
+										{
+											MountPath: "/data",
+											Name:      "data",
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+var _ = framework.KubeDescribe("Generated release_1_5 clientset", func() {
+	f := framework.NewDefaultFramework("clientset")
+	It("should create v2alpha1 scheduleJobs, delete scheduleJobs, watch scheduleJobs", func() {
```

Comment:
This is a good specific test to watch for regression in the 1.5 clientset.  Nice job.

A more general test of this functionality would be good eventually, but seems like a big project.